.github/workflows/build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           bash <(curl -s https://codecov.io/bash) -c -F tools -f '!*.txt'
 
   release-github:
-    if: github.repository == 'nh13/conda-env-builder' && github.ref == 'refs/heads/main'
+    if: github.repository == 'conda-incubator/conda-env-builder' && github.ref == 'refs/heads/main'
```

Comment:
we will need a secret named `REPO_ACCESS_TOKEN` to make this work

Reply:
Ah, it's still there FWIW!

Reply:
Phew!